### ISSUE TITLE
Present project at CivicTech Toronto

### ISSUE THREAD

**Issue by benhylau** [tomeshnet/toronto-community-network#76]
<sup>_This initial comment is collaborative and open to modification by all._</sup>

## Task Summary

🎟️ **Re-ticketed from:** #
📅 **Due date:** 29 September 2020
🎯 **Success criteria:** Present project at CivicTech Toronto and have a plan to onboard people ongoing.

## Civic Tech Presentation Topics To Cover
- Who is Toronto Mesh
- What we are trying to accomplish
- What we have accomplished
- What we need help with:
  - Network literacy workshop with TPL
  - Social media help
  - Design help for website & branding
  - Grant proposal writing (ISOC grants, Beyond the Net, CIRA)
  - Ambassadors to help spread the good word of ~~mesh networking~~ community networks (edit by @benhylau)
  - People who want to connect to the network!

## To Do

- [x] Complete #83 
- [x] Revamp slides
- [x] Share with others for review
- [x] Present it!

**Comment by makeworld-the-better-one** [comment]
#83 must be completed before the presentation happens.

**Comment by darkdrgn2k** [comment]
> #83 must be completed before the presentation happens.

@makeworld-the-better-one 

Please add it as a todo in the first post.  Otherwise we will never find it in the thread :+1:

**Comment by benhylau** [comment]
@makeworld-the-better-one @Shrinks99 do you guys wanna present on sept 29?

**Comment by makeworld-the-better-one** [comment]
That sounds good to me.

**Comment by therealitycorp** [comment]
Hey everyone, Sam from Civic Tech (speaker booking), psyched to hear what you're up to! I'll pencil you in for September 29, please confirm as soon as you can :)

Let me know if you have any questions about the format / etc. Thanks!

**Comment by Shrinks99** [comment]
Hi @therealitycorp!  September 29th sounds good to me, looks like there's a date attached to our user on-boarding process haha.

**Comment by therealitycorp** [comment]
@Shrinks99 Amazing :)

10 minutes is the target, and then ~15 minutes for Q&A. We're currently workshopping the format, so I'll give you a heads up if this changes in the next month. If there are any questions or topics you want to work through with the CTTO community, you can also host a "break out" room after the talk.

Things I'll need from you: (no rush!)
- [ ] Talk title
- [ ] Talk description
- [ ] Short organization bio/description & social media links
- [ ] Speaker bio(s) & social media links
- [ ] Speaker photos

Thanks everyone!

**Comment by benhylau** [comment]
Hey @Shrinks99 @makeworld-the-better-one I would like to post this on https://tomesh.net/events/ and also schedule an in-person welcome/install party ~one week after. The event will go something like this, perhaps you can extend the invite in your presentation?

>We are holding an in-person welcome BBQ on October 10 Saturday, where members from different working groups will be present, and the Network working group will show us how to configure hardware for those interested in helping with upcoming supernode deployments.
>
>There will be a limit on number of attendees due to COVID-19 precautions. Please email operations@tomesh.net if you would like to join us!

Can you let me know event info including call time and link, and a brief description once those are confirmed? I'd like to post the events on our website soon :)

cc. @YurkoWasHere @TimTor @Pedro-on-a-bike this is the install party we discussed. Oct 10 works for you guys?

**Comment by darkdrgn2k** [comment]
24th is a meet and greet project update
So any additional QandA or any in depth questisons can bee deferred to Thursday's meeting.

**Comment by therealitycorp** [comment]
Hey everyone—Sept 29 is almost a week away! Can you share asap:

- [ ]  Talk title
- [ ]  Talk description
- [ ]  Short organization bio/description & social media links
- [ ]  Speaker bio(s) & social media links
- [ ]  Speaker photos (or a photo we can use for the event header)

Thanks!

**Comment by benhylau** [comment]
@Shrinks99 as requested
```
join,https://cloud.tomesh.net/apps/forms/3eJKtBKQ3m4Br27B
wg,https://github.com/tomeshnet/toronto-community-network/blob/master/README.md
contact,https://cloud.tomesh.net/apps/onlyoffice/5150?filePath=%2Ftomeshnet-trusted%2Fcontact-management.xlsx
contacts,https://cloud.tomesh.net/apps/onlyoffice/5150?filePath=%2Ftomeshnet-trusted%2Fcontact-management.xlsx
calendar,https://calendar.google.com/calendar/embed?src=rtir6741jis1vovitm068jfob0%40group.calendar.google.com
```

**Comment by Shrinks99** [comment]
@therealitycorp Sorry this has taken so long!  Here's the requested information:

**Title:** All About The Toronto Community Network

**Description:** Building Toronto's First Mesh Network

**Organization Bio / Links:** Toronto Mesh helps communities create better networks with open source and peer-to-peer technologies that promote digital literacy and privacy.
Twitter: @tomeshnet
Website: https://tomesh.net

**Speaker Bio's / Links:**
Henry Wilkinson is a fourth year design student at York University & Sheridan college, he is currently on communications team of Toronto Mesh working on the Toronto Community Network.
Twitter: @Shrinks99
Website: https://wilkinson.graphics

![Henry Wilkinson 1024](https://user-images.githubusercontent.com/5672810/93726799-f9e55580-fb85-11ea-9c76-ca046cb188cf.jpg)


Cole AC is a programmer in his graduating year of high school and has participated in Toronto Mesh for several years. He is currently working on the communications and networking teams for the Toronto Community Network.
GitHub: @makeworld-the-better-one

![eAyVBBwIGHsAhEpHWYUatcoh](https://user-images.githubusercontent.com/5672810/93726805-049fea80-fb86-11ea-8ad4-6e93ccc9129b.jpg)

**Comment by therealitycorp** [comment]
Amazing, thanks! Description is perfect.

Feel free to share the event link, or let me know if I can edit it for you in any way:
https://www.meetup.com/Civic-Tech-Toronto/events/272619247/

**Comment by therealitycorp** [comment]
@Shrinks99 @makeworld-the-better-one 

Two last questions—

- We typically record the talk and Q&A portion of the evening to share later on our YouTube channel—are you comfortable being recorded?
- Can you share a link to your slides with me a day prior, so I can send them out to the attendees? (for accessibility purposes)

**Comment by Shrinks99** [comment]
We're both good with being recorded!  What's the best format for slides, PDF?  I will be sure to tag all our images with alt text.

**Comment by therealitycorp** [comment]
PDF, .ppt, or Google Slides are all good—and thanks for thinking of the alt text :) You can send them to sam.civictech at gmail whenever you're ready!
